### DISCUSSION TITLE
Adding Characters

### DISCUSSION THREAD

**Discussion by ghost** [FunkinCrew/Funkin#1526] [Q&A]
Got another problem :(

When I try to build the game with a new character, it says; "source/HealthIcon.hx:37: characters 3-12 : Missing ;"

Please if you can, send a solution, thanks!

**Comment by DamianBossPL** [comment]
Try following this tutorial:
https://youtu.be/jNwkEJ3-rXA

**Comment by ghost** [comment]
I was using that tutorial but it still won't work :(

**Comment by 4EverNoobys** [comment]
Isn't it clear that it said you're missing a ";"

**Comment by R-code-0** [comment]
You need to add ; to end of line 37.
